packages/block-library/src/post-time-to-read/variations.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { clock } from '@wordpress/icons';
+
+const variations = [
+	{
+		name: 'time-to-read',
+		title: __( 'Time to Read' ),
+		description: __( 'Show minutes required to finish reading the post.' ),
+		attributes: {
+			displayMode: 'time',
+		},
+		scope: [ 'block', 'inserter', 'transform' ],
+		isActive: ( blockAttributes ) =>
+			blockAttributes?.displayMode === 'time',
+		icon: clock,
```

Comment:
```suggestion
		icon: clock,
		isDefault: true,
```

Prevent multiple Time to Read blocks from appearing

<img width="1000" height="309" alt="Image" src="https://github.com/user-attachments/assets/a94b9390-383c-4d54-b1a1-e65da4f85f30" />